Modify note and a step in the SCA enabled case (#1967)
theforeman/foreman-documentation: guides/common/modules/proc_attaching-satellite-infrastructure-subscription.adoc

[NOTE]
====

> This step is *optional* if you have SCA enabled in {Project}.

Reviewer: ```suggestion
This step is *optional* if you have SCA enabled on {Project}.
```

[NOTE]
====

> This step is *optional* if you have SCA enabled on {Project}.

Reviewer 2: Like I said in the bz, I feel like the word "optional" is misleading, since with SCA enabled sub-man won't even _let_ you attach a subscription.

Author: Reworded :+1:

[NOTE]
====

> This step is *optional* if you have SCA enabled in the {ProjectName}.

Reviewer 3: ```suggestion
This step is *optional* if you have SCA enabled in {Project}.
```

> For more information, see the Red{nbsp}Hat Knowledgebase solution https://access.redhat.com/solutions/3382781[{Project} Infrastructure Subscriptions MCT3718 MCT3719].

Reviewer 3: ```suggestion
For more information, see https://access.redhat.com/solutions/3382781[{Project} Infrastructure Subscriptions MCT3718 MCT3719] in the _Red{nbsp}Hat Knowledgebase_.
```

. Make a note of the subscription Pool ID.
Your subscription Pool ID is different from the example provided.

. Attach the {Project} Infrastructure subscription to the base operating system that your {ProductName} is running on.

> This step can be skipped if SCA is enabled for {Project}.

Reviewer 3: ```suggestion
If SCA is enabled for {Project}, you can skip this step:
```

Should this be "on {ProjectServer}" instead of "for Project"?

Author: Agree. Addressed.

Your subscription Pool ID is different from the example provided.

. Attach the {Project} Infrastructure subscription to the base operating system that your {ProductName} is running on.

> If SCA is enabled on {ProjectServer}, you can skip this step:

Reviewer: Keeping it in consistency with the note.
```suggestion
If you have SCA enabled on {Project}, you can skip this step:
```
